crates/c-api/src/ref.rs

```diff
@@ -181,3 +185,158 @@ wasmtime_c_api_macros::declare_ref!(wasm_foreign_t);
 pub extern "C" fn wasm_foreign_new(_store: &crate::wasm_store_t) -> Box<wasm_foreign_t> {
     abort("wasm_foreign_new")
 }
+
+pub type wasmtime_anyref_t = ManuallyDrop<ManuallyRooted<AnyRef>>;
+pub type wasmtime_externref_t = ManuallyDrop<ManuallyRooted<ExternRef>>;
```

Comment:
Mind adding a comment here as to what the `ManuallyDrop` is doing? It seems related to the destructor of `wasmtime_val_t` but I couldn't quite figure it out

Reply:
This is a leftover from the old externref support, and I don't think we actually need it anymore. I'll remove it.